libethereum/Account.cpp

```diff
@@ -88,11 +90,12 @@ namespace
 	string const c_balance = "balance";
 	string const c_nonce = "nonce";
 	string const c_code = "code";
+	string const c_file = "file";  ///< A file containg a code as bytes.
 	string const c_storage = "storage";
 	string const c_shouldnotexist = "shouldnotexist";
 	string const c_precompiled = "precompiled";
 	std::set<string> const c_knownAccountFields = {
-		c_wei, c_finney, c_balance, c_nonce, c_code, c_storage, c_shouldnotexist,
+		c_wei, c_finney, c_balance, c_nonce, c_code, c_file, c_storage, c_shouldnotexist,
```

Comment:
you put path to the file with code as an attribute of account object?

Reply:
Yes.

Reply:
I thought the idea was to load code from file upon loading genesis.json
why would we need to change logic of Accont.h ?

Reply:
Load a code from a file what do what with it?

Reply:
so as I understand currently account's code is loaded from genesis.
could we change it so that upon loading that genesis account code will just be read from external file? other then that it is the same genesis config file loading.

Reply:
I don't understand

Reply:
in jsonToAccountMap at
if (o[c_code].type() == json_spirit::str_type)
{
   if (o[c_code].get_str().substr(0,2) == "//")
        read code from file in this link and parse it.
}

I thought we could just do it that way.

Reply:
I think it is better making it more explicit than hiding it via special expressions.